frontend/src/components/pages/TourDeFoos/TourDeFoos.tsx

```diff
@@ -1,77 +1,71 @@
+import {Button, MuiThemeProvider} from "@material-ui/core";
+import {Centered} from "../../atoms/Shared/Centered";
+import {BrowserRouter as Router, Switch, Route} from "react-router-dom";
 import React from "react";
-import { Button, MuiThemeProvider } from "@material-ui/core";
-import { unstable_createMuiStrictModeTheme as createMuiTheme } from "@material-ui/core";
-import { grey } from "@material-ui/core/colors";
-import { Centered } from "../../atoms/Shared/Centered";
-import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
 import {
-  PATH_FOR_CREATING_NEW_TOURNAMENT_VIEW,
-  PATH_FOR_LOGIN_VIEW,
-  PATH_FOR_PLAYER_PROFILE_CREATION_VIEW,
-  PATH_FOR_PLAYERS_PROFILES_VIEW,
-  PATH_FOR_HOME_VIEW,
-  PATH_FOR_TOURNAMENT_REGISTRATIONS_VIEW,
-  PATH_FOR_TOURNAMENTS_SELECTION_VIEW,
+    PATH_FOR_CREATING_NEW_TOURNAMENT_VIEW,
+    PATH_FOR_LOGIN_VIEW,
+    PATH_FOR_PLAYER_PROFILE_CREATION_VIEW,
+    PATH_FOR_PLAYERS_PROFILES_VIEW,
+    PATH_FOR_HOME_VIEW,
+    PATH_FOR_TOURNAMENT_REGISTRATIONS_VIEW,
+    PATH_FOR_TOURNAMENTS_SELECTION_VIEW, PATH_FOR_PLAYER_MATCHES_VIEW,
 } from "../../atoms/constants/paths";
-import { HomeMenu } from "../HomeMenu/HomeMenu";
-import { TournamentRegistrations } from "../TournamentRegistrations";
+
+import {MatchesList} from "../../organisms/MatchesList";
+import {THEME} from "../../atoms/constants/ThemeMUI";
 import Footer from "../../organisms/Footer/Footer";
-import { VerticalSpace } from "../../atoms/Shared/VerticalSpace";
+import {VerticalSpace} from "../../atoms/Shared/VerticalSpace";
 import Header from '../../organisms/Header/Header';
+import {TournamentRegistrations} from "../TournamentRegistrations";
+import {HomeMenu} from "../HomeMenu/HomeMenu";
 
-const theme = createMuiTheme({
-  palette: {
-    primary: {
-      main: grey[800],
-      contrastText: "#E3E152",
-    },
-    secondary: {
-      main: "#FFD100",
-      contrastText: "#000000",
-    },
-  },
-});
```

Comment:
Tylko tu zmiana w tym pliku

- reszta to format bo był niesformatowany

Reply:
Dobrze byłoby dodać sprawdzanie formata do GitHub Actions w takim razie :)

Reply:
![image](https://user-images.githubusercontent.com/41823689/113932013-70c8ff80-97f3-11eb-8fa1-4a4749b5e261.png)
Hmmm? Tak mogę ?
(następny PR?)